### ISSUE TITLE
XPM - Gerar arquivos com texto completo traduzido para PMC

### ISSUE THREAD

**Issue by RPostalli** [scieloorg/pcp#13]
O PMC já aceita textos traduzidos, portanto o XPM deve manter os idiomas traduzidos no XML PMC em `<sub-article>`.

As regras para XML PMC Multi-Language Articles pode ser observado no link: https://www.ncbi.nlm.nih.gov/pmc/pmcdoc/tagging-guidelines/article/dobs.html#dob-multi-lang

O PMC aguarda o envio de uma pequena amostra com os XMLs PMC em mais de um idioma para análise.

Solicitar testes para @nathaliarosa

**Comment by nathaliarosa** [comment]
@robertatakenaka

O XPM está gerando o `<sub-article>` das traduções no XML corretamente, mas notamos que, quando o idioma em `<article>` **não** é o inglês, o XPM exclui os PDFs e imagens do pacote, deixando apenas o XML.

Isso acontece para artigos que tem ou não tradução e o idioma em `<article>` não é o inglês.

**Exemplos:**

Artigo somente em português - pacote SciELO
![scielo](https://github.com/scieloorg/pcp/assets/105010471/9ed420f7-34c7-4c7e-a74f-900ecb8880a8)
[0104-5970-hcsm-31-e2024021_pacoteScielo.zip](https://github.com/user-attachments/files/15791928/0104-5970-hcsm-31-e2024021_pacoteScielo.zip)

Artigo somente em português - pacote PMC
![pmc](https://github.com/scieloorg/pcp/assets/105010471/e71a43f4-73ef-4f04-be6b-4b9fb15b939f)
[0104-5970-hcsm-31-e2024021_pacotePMC.zip](https://github.com/user-attachments/files/15791930/0104-5970-hcsm-31-e2024021_pacotePMC.zip)

Artigo em português com tradução em inglês - pacote SciELO
![scielo_t](https://github.com/scieloorg/pcp/assets/105010471/cdde67ca-72f9-49d9-9123-8f62e6868989)
[1678-4464-csp-40-05-PT169123_pacoteScielo.zip](https://github.com/user-attachments/files/15791965/1678-4464-csp-40-05-PT169123_pacoteScielo.zip)

Artigo em português com tradução em inglês - pacote PMC
![pmc_t](https://github.com/scieloorg/pcp/assets/105010471/78ec8756-4cdc-4c7a-87bb-13948c2b1531)
[1678-4464-csp-40-05-PT169123_pacotePMC.zip](https://github.com/user-attachments/files/15792004/1678-4464-csp-40-05-PT169123_pacotePMC.zip)

O XPM deve gerar o pacote com todos os arquivos que estão nele (XML, PDF, imagens, etc), independente do idioma principal do artigo, ou seja, do idioma marcado em `<article>`.

PSI @RPostalli

**Comment by RPostalli** [comment]
@robertatakenaka enquanto você aplica essa correção mencionada pela @nathaliarosa, eu já envie hoje 11/06 ao PMC a amostra com os artigos em inglês com tradução em outros idiomas e aguardarei análise do XML PMC gerado. Te informo assim que possível.

**Comment by robertatakenaka** [comment]
@nathaliarosa se o article é em inglês todas as imagens etc entram no pacote?
Não será um desenvolvimento rápido fazer esta alteração. Estimativa: julho

**Comment by nathaliarosa** [comment]
@robertatakenaka sim, quando o `<article>` é em inglês (tendo ou não traduções) todos os PDFs e imagens são gerados e entram no pacote. Quando o `<article>` está em português ou espanhol (tendo ou não traduções) só é gerado o XML, os outros arquivos são excluídos do pacote.

**Comment by nathaliarosa** [comment]
@robertatakenaka

Após a mudança para o XPM gerar o `<sub-article>` nos XMLs PMC, vimos que a tag `<mixed-citation>` está sendo gerada nas referências em todos os XMLs PMC.

**Descrição do problema**
o elemento `<mixed-citation>` está sendo gerado dentro de `<ref>` em todos os XMLs PMC.

**Passos para reproduzir o problema**
1. Acessar o XPM na WinTS.
2. Selecionar um lote que possua arquivo XML com referências.
3. Gerar o pacote PMC no XPM.
4. Observe que os XMLs gerados pelo XPM possuem a tag `<mixed-citation>` dentro de `<ref>`.

**Comportamento esperado**
Em `<ref>` é esperado que o XPM retire automaticamente o elemento `<mixed-citation>`, mantendo somente a marcação detalhada dentro de `<element-citation>`.

**Screenshots**

Print de XML com erro - com a tag `<mixed-citation>`
<img width="311" alt="image" src="https://github.com/scieloorg/pcp/assets/105010471/74f970d9-65e0-47ca-94f6-bacf213b8efe">

Print de XML mostrando o comportamento esperado pelo XPM - sem a tag `<mixed-citation>`, apenas a marcação detalhada dentro de `<element-citation>`
![image](https://github.com/scieloorg/pcp/assets/105010471/f7ae0e02-668d-41fd-8240-856c0485a83b)

PSI @RPostalli
